sdl_ios/SmartDeviceLink/SDLIAPTransport.m

```diff
@@ -295,16 +311,15 @@ - (void)openSession {
         } else {
             if ([self.protocolString isEqualToString:CONTROL_PROTOCOL_STRING]) {
```

Comment:
Can you please add more context to this statement?
Which `if statement`?  Why do you feel it is no longer needed?